examples/stable-diffusion/text_to_image_generation.py

```diff
@@ -121,9 +121,25 @@ def main():
         ),
     )
     parser.add_argument("--bf16", action="store_true", help="Whether to perform generation in bf16 precision.")
+    parser.add_argument(
+        "--ldm3d", action="store_true", help="Use LDM3D to generate an image and a depth map from a given text prompt."
+    )
+    parser.add_argument(
+        "--ldm3d_model_name_or_path",
+        default="Intel/ldm3d-4c",
+        type=str,
+        help="Path to pre-trained model",
+    )
```

Comment:
I would rather remove this argument and add the following:
```py
if args.ldm3d and args.model_name_or_path == "runwayml/stable-diffusion-v1-5":
    args.model_name_or_path = "Intel/ldm3d-4c"
```
That way we make sure that we override the default model if it is not specified by the user.

Reply:
Great idea!
I pushed the updated version

Reply:
Yes, no worries :)
It failed because the CI must be manually triggered by a repo maintainer when the PR comes from a fork. I just did it so it should finish soon.